### PR TITLE
Task #5253 - User view detail book

### PR DESCRIPTION
**Pull request by taylq**
https://edu-redmine.framgia.vn/issues/5253?issue_count=30&issue_position=7&next_issue_id=5250&prev_issue_id=5254

### CODE REVIEW

**Comment by taylq** [comment]
ready

**Comment by taylq** [comment]
ready

File: app/views/books/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<div class="row">
+  <div class="col-sm-2">
+    <%= image_tag @book.image if @book.image? %>
+    <h5><%= @book.name %></h5>
+    <h5><%= @book.status %></h5>
+    <h3><%= t "books.publisher" %></h3>
+    <%= @book.publisher.name %>
```
Comment (huongnguyenmta) [inline app/views/books/show.html.erb]: dùng `delegate`

File: app/views/books/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    <%= @book.publisher.address %>
+  </div>
+  <div class="col-sm-8" style="text-align: center">
+    <h2><%= @book.name %></h2>
+    <h6><%= @book.content %></h6>
+  </div>
+  <div class="col-sm-2">
+    <h3><%= t "books.author" %></h3>
+    <% @book.book_authors.each do |ba| %>
+      <h4><%= ba.author.name %></h4>
```
Comment (huongnguyenmta) [inline app/views/books/show.html.erb]: dùng delegate

File: db/seeds.rb [hunk 3]
```diff
@@ -31,8 +31,8 @@
 end
 
 puts "7. Seeding Book_Author"
-Author.all.each do |author|
-  Book.all.each do |book|
-    BookAuthor.create!(book_id: book.id, author_id: author.id)
+3.times do
+Book.all.each do |book|
```
Comment (huongnguyenmta) [inline db/seeds.rb]: indent

**Comment by taylq** [comment]
ready

**Pull request merged**